src/dotty/language.scala

```diff
@@ -4,6 +4,16 @@ object language {
 
   class Feature
 
+  /** Enable scala.Dynamic */
```

Comment:
Can we leave everything in scala.language? My latest PR #1550 removed dotty.language completely. 

Otherwise LGTM